en/editor/publish/publish-bytedance-mini-game.md

```diff
@@ -0,0 +1,62 @@
+# Publish to ByteDance Mini Games
+
+ByteDance mini-game is developed based on the ByteDance full products, which does not require users to download, and is a brand-new game type that can be played on tap.
+
+The game submission, review and release process of the mini-games need to comply with the requirements and standard processes of the Byte official team. For specific information, please refer to the link at the end of the article.
+
+## Publish ByteDance with Cocos Creator 3D
+
+1. Download **ByteDance DevTools** on [ByteDance Official Document](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/developer-instrument-update-and-download).
```

Comment:
Download **ByteDance DevTools** on [ByteDance Official Document]
-->
Download the **ByteDance DevTools** from the  [ByteDance Official Website]

Reply:
This change still needs to be done.